qiling/os/windows/dlls/msvcrt.py

```diff
@@ -498,6 +469,42 @@ def hook_malloc(ql: Qiling, address: int, params):
 
     return ql.os.heap.alloc(size)
 
+def __realloc(ql: Qiling, address: int, params):
```

Comment:
Isn't this pretty much the same as heap realloc?
Consider refactoring them both in a way they could share common code.